global.json

```diff
@@ -15,6 +15,7 @@
   "msbuild-sdks": {
     "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.24102.4",
     "Microsoft.DotNet.Helix.Sdk": "9.0.0-beta.24102.4",
+    "Microsoft.Build.NoTargets": "3.7.0",
     "Microsoft.Build.Traversal": "3.2.0",
     "Microsoft.SourceLink.GitHub": "1.1.0-beta-20206-02",
     "Microsoft.SourceLink.Common": "1.1.0-beta-20206-02"
```

Comment:
For a later clean-up round, these can be removed as sourcelink is now part of the .NET SDK.